### ISSUE TITLE
Disabling block

### ISSUE THREAD

**Issue by nevergone** [microsoft/pxt#10042]
It would be useful if blocks could be disabled or deactivated in the block editor.
They would not be deleted, they would just appear differently. It would be similar to commenting lines of code in source code.

**Comment by BrightKn1ght** [comment]
I also would be interested in this described feature.

**Comment by WueCoding** [comment]
I am also interested!

**Comment by THEb0nny** [comment]
This is a really good idea, but is there something like this in the parent Google Blockly?

**Comment by BrightKn1ght** [comment]
> This is a really good idea, but is there something like this in the parent Google Blockly?

I quickly checked blockly 10.4.1. When you right click a block there's a "Disable" option in the context menu. So seems there's already something there. The difficulty will be to modify the transpiler to skip that block.

**Comment by THEb0nny** [comment]
Indeed, I looked, there the blocks are switched off and they become translucent.
@riknoll is it difficult to implement now?

**Comment by THEb0nny** [comment]
@riknoll I'm interested in whether it's possible to disable a block without removing it from the algorithm with other blocks. In Blockly, right-clicking on a block opens a menu with an option to disable the block.

**Comment by nevergone** [comment]
@riknoll Could you make this feature available directly?

**Comment by YourCodeShepherd** [comment]
Would be a quite useful feature, to be more flexible and do quicker changes to the program.

**Comment by THEb0nny** [comment]
Maybe the comment block proposal will be included in the development plans?

**Comment by nevergone** [comment]
And now?

**Comment by nevergone** [comment]
Hello, how can we help to complete this?
